packages/quizzes-dashboard/components/AnswerListWrapper.tsx

```diff
@@ -19,19 +19,15 @@ export const Paginator = styled(Pagination)`
 const StyledSkeleton = styled(Skeleton)`
   margin-bottom: 1rem;
 `
-const SkeletonContainer = styled.div`
-  display: flex;
-  width: 100%;
-  justify-content: center;
-  margin-bottom: 1rem;
-`
 
 interface WrapperProps {
   quizId: string
   size: number
   order: string
   filterparameters: string[]
   expandAll: boolean
+  page: number
+  handleChange: any
```

Comment:
Can we have a more specific type annotation for this? Using `any` is fine if the variable can actually be anything or the type of the variable is difficult to express.